### DISCUSSION TITLE
Adding the User/Access Key used to login to the Logs within the same "new listener" log entry

### DISCUSSION THREAD

**Discussion by sstack124** [chuot/rdio-scanner#226] [Ideas]
Would it be possible to add the User/Access Key used to login in the Logs within the same "new listener" log entry? If access keys were not configured, the way the log is written for "new listener" would be fine.
Something like this:
new listener (JimSmith) from ip 192.168.1.12

**Comment by chuot** [comment]
It's totally feasible, but It will require some changes to the code. Keep an eye on the upcoming v6.5.0, I may be able to add/test that feature in time before the release.
